### PR TITLE
Codesink 04 06 17

### PR DESCRIPTION
**Pull request by jkopel**
Brings field_collections to code parity with paragraphs.
Fixes all the tests (he said hopefully).

### CODE REVIEW

**Comment by jkopel** [comment]
I meant simple not unit.

File: test/Features/contentTranslation/ReferencedEntities.feature [hunk 1]
```diff
@@ -42,7 +42,6 @@ Feature: Referenced Entity Translation
     When I click "View"
     And I click "Français"
     Then I should see "fr host page title"
-    Then I Expect You To Die
```
Comment (iamEAP) [inline test/Features/contentTranslation/ReferencedEntities.feature:45]: 🔪

**Comment by iamEAP** [comment]
LGTM

File: test/Features/contentTranslation/ReferencedEntities.feature (hunk 1 above)
Comment (jkopel) [reply test/Features/contentTranslation/ReferencedEntities.feature:45]: 🐞  🔫  ☠️

**Pull request merged**